Tests/HaCWebsiteLibTests/HaCWebsiteLibTests.swift

```diff
@@ -2,16 +2,94 @@ import XCTest
 @testable import HaCWebsiteLib
 
 class HaCWebsiteLibTests: XCTestCase {
```

Comment:
👍 I was a little shaky on the structure of these test cases but have now structured it more logically